src/test/java/com/puppycrawl/tools/checkstyle/checks/design/OneTopLevelClassCheckTest.java

```diff
@@ -132,43 +132,66 @@ public void testFileWithThreeTopLevelEnum() throws Exception {
         final DefaultConfiguration checkConfig =
                 createModuleConfig(OneTopLevelClassCheck.class);
         final String[] expected = {
-            "3: " + getCheckMessage(MSG_KEY, "InputOneTopLevelClassEnum2inner1"),
-            "11: " + getCheckMessage(MSG_KEY, "InputOneTopLevelClassEnum2inner2"),
+            "3:1: " + getCheckMessage(MSG_KEY, "InputOneTopLevelClassEnum2inner1"),
+            "11:1: " + getCheckMessage(MSG_KEY, "InputOneTopLevelClassEnum2inner2"),
         };
         verify(checkConfig, getPath("InputOneTopLevelClassEnum2.java"), expected);
     }
 
     @Test
     public void testFileWithFewTopLevelClasses() throws Exception {
         final DefaultConfiguration checkConfig =
-            createModuleConfig(OneTopLevelClassCheck.class);
+                createModuleConfig(OneTopLevelClassCheck.class);
```

Comment:
Undo unnecessary changes like this. I see others in the same diff.

Reply:
Fixed (I think I got all of them)